presto-hive/src/test/java/com/facebook/presto/hive/AbstractTestHiveClient.java

```diff
@@ -2670,29 +2670,49 @@ public void testCreateTableUnsupportedType()
     @Test
     public void testCreateBucketedTemporaryTable()
             throws Exception
+    {
+        testCreateBucketedTemporaryTable(newSession());
+    }
+
+    protected void testCreateBucketedTemporaryTable(ConnectorSession session)
```

Comment:
Maybe I'm missing something, but it seems that this method is not needed and we could simply do
```
    @Test
    public void testCreateBucketedTemporaryTable()
            throws Exception
    {
        ConnectorSession session = newSession();
        testCreateBucketedTemporaryTable(session, true);
        testCreateBucketedTemporaryTable(session, false);
    }
```

Reply:
I'm overriding this method in the internal connector test suite